redfish-exporter/listener.go

```diff
@@ -135,17 +157,16 @@ func handleConnection(AppConfig Config, conn net.Conn, eventCount *int, dataBuff
 			break
 		}
 
-		err = processRequest(AppConfig, conn, req, eventCount, dataBuffer, remote)
+		err = processRequest(AppConfig, conn, req, eventCount, dataBuffer)
```

Comment:
When we scale to multiples of servers listened to, are we sure that every server uses the same SLURM endpoint? This remote was being passed so we can then map the event to a server and from the server object find the correct slurm endpoint. 
If we are certain that all servers will always be handled by the same slurm endpoint, then this is not needed, and OK to remove.

Reply:
Or, I see we are extracting it again below.. never mind :D

Reply:
Yeah, I relocated it closer to its usage since the handleConnection didn't need it.